src/lib.rs

```diff
@@ -1,6 +1,12 @@
+#![no_std]
```

Comment:
Will this work with `std` as well? I thought in the past crates needed to have a separate Cargo feature for this mode?

Reply:
Yes it works with std as well.

Reply:
You only need an std feature for enabling std-only functionality.

Reply:
Thanks.